modules/bmo-about-the-hostupdatepolicy-resource.adoc

```diff
@@ -0,0 +1,34 @@
+// This is included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="bmo-about-the-hostupdatepolicy-resource_{context}"]
+= About the HostUpdatePolicy resource
+
+You can use the `HostUpdatePolicy` resource to enable or disable applying live updates to the firmware settings, BMC settings, or BIOS settings of each bare-metal host. By default, the operator disables live updates to already provisioned bare-metal hosts by default.
+
+.The HostUpdatePolicy spec
```

Comment:
You are allowed to use backticks in labels. There is a rule against using them in headings, but specific object references in labels should use backticks. See the repo guidelines on API object references: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

This feedback applies throughout the PR, I won't mention it in other instances.